frontend/src/components/PatientForm.js

```diff
@@ -1,26 +1,23 @@
-import React from "react";
-//import TextField from '@material-ui/core/TextField';
-import {Button} from "@material-ui/core";
+import React from 'react';
+// import TextField from '@material-ui/core/TextField';
+import { Button } from '@material-ui/core';
 
+const handleSubmit = (e) => {
+  console.log(e);
+  e.preventDefault();
+};
 
-const handleSubmit = e => {
-    console.log(e)
-    e.preventDefault()
+/* eslint-disable react/no-this-in-sfc */
+export default function PatientForm(patient) { // eslint-disable-line no-unused-vars
```

Comment:
"patient" blir ikke brukt, og "this" bør ikke brukes da det ikke er en klasse(?)

Reply:
PatientForm.js blir generelt ikke brukt, da dette var et forsøk på å modularisere som vi gikk bort ifra. Tror egentlig vi bare kan slette denne filen.